app/src/main/java/com/yacgroup/yacguide/utils/DataUId.kt

```diff
@@ -15,16 +15,6 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  */
 
-package com.yacgroup.yacguide.network
+package com.yacgroup.yacguide.utils
 
-enum class RequestType {
-    COUNTRY_DATA,
-    REGION_DATA,
-    SECTOR_DATA,
-    ROCK_DATA,
-    ROUTE_DATA,
-    REGION_COMMENTS,
-    SECTOR_COMMENTS
-}
-
-class NetworkRequestUId(val type: RequestType, val id: Int, val name: String = "")
+class DataUId(val id: Int, val name: String)
```

Comment:
I still have this opinion but regarding `DateUId` struct, I agree with you that this name is pretty indescriptive. However, I could not find a better name since for the update and parser, it's only data to be identfied.

Reply:
I already struggle with the name `UId`. What exact means `UId` in our case?

Reply:
Well, it is some sort of identification while an id is not enough since this is used for all kinds of elements (regions, sectors, ...), of an id would be ambiguous. `UId` is not so nice either since there is another `UId` class in java lib. Maybe `packageUId` because we update somehow packages? So a package could be RegionData for a region or SectorData for a sector etc.

Reply:
I will call it `ClimbingObjectUId`. This is clearly understandable and I can reuse the already existing ClimbingObject struct fir this.